src/test/java/unstable/KeyboardShortcutsConfigTest.java

```diff
@@ -94,25 +83,11 @@ public void repeatedKeySpecified() {
         keyboardShortcuts.remove("MARK_AS_UNREAD");
         keyboardShortcuts.put("MARK_AS_UNREAD", "E");
         testPref.setKeyboardShortcuts(keyboardShortcuts);
-        reloadPrefs(testPref);
         Platform.runLater(() -> KeyboardShortcuts.loadKeyboardShortcuts(testPref));
         waitUntilNodeAppears(hasText("Invalid key specified for MARK_AS_UNREAD" +
             " or it has already been used for some other shortcut. "));
         click("Use default key");
-        reloadPrefs(testPref);
         assertEquals(KeyboardShortcuts.getDefaultKeyboardShortcuts().get("MARK_AS_UNREAD"),
             testPref.getKeyboardShortcuts().get("MARK_AS_UNREAD"));
     }
-
-    private static void reloadPrefs(Preferences prefs) {
```

Comment:
You're right. I didn't fully understand it at first. I'll adapt the old method to our new way of doing things